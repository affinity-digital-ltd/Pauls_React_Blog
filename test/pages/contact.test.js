import Contact from '../../pages/contact'

describe('<Contact />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <Contact />
    )

    expect(component).toMatchSnapshot()
  })
})
