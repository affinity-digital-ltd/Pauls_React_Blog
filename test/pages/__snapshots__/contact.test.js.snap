// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contact /> it renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Contact />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <div
        className="o-contact__container"
      >
        <h2>
          Contact
        </h2>
        <div
          className="c-header-divider"
        />
        <section
          className="c-contact-details"
        >
          <h3>
            Connect with me
          </h3>
          <p>
            I am conveniently situated between Wakefield and Doncaster in Fitzwilliam, Wakefield and have access to the train station that connects to Leeds and Sheffield.
          </p>
          <p>
            I will be actively looking for work as a web developer as soon as I finish my course at {Pro}Coders.
          </p>
          <p>
            Feel free to contact me with any comments or inquires using the contact form or connect with me on social media.
          </p>
          <ul>
            <li>
              <a
                href="https://twitter.com/paulbrighton_"
              >
                Twitter
              </a>
            </li>
            <li>
              <a
                href="https://www.facebook.com/paul.brighton.10"
              >
                Facebook
              </a>
            </li>
            <li>
              <a
                href="https://www.pinterest.co.uk/paulbrightonfda/"
              >
                Pinterest
              </a>
            </li>
            <li>
              <a
                href="https://www.linkedin.com/in/paulbrighton/"
              >
                LinkedIn
              </a>
            </li>
          </ul>
        </section>
        <ContactForm />
      </div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            Contact
          </h2>,
          <div
            className="c-header-divider"
          />,
          <section
            className="c-contact-details"
          >
            <h3>
              Connect with me
            </h3>
            <p>
              I am conveniently situated between Wakefield and Doncaster in Fitzwilliam, Wakefield and have access to the train station that connects to Leeds and Sheffield.
            </p>
            <p>
              I will be actively looking for work as a web developer as soon as I finish my course at {Pro}Coders.
            </p>
            <p>
              Feel free to contact me with any comments or inquires using the contact form or connect with me on social media.
            </p>
            <ul>
              <li>
                <a
                  href="https://twitter.com/paulbrighton_"
                >
                  Twitter
                </a>
              </li>
              <li>
                <a
                  href="https://www.facebook.com/paul.brighton.10"
                >
                  Facebook
                </a>
              </li>
              <li>
                <a
                  href="https://www.pinterest.co.uk/paulbrightonfda/"
                >
                  Pinterest
                </a>
              </li>
              <li>
                <a
                  href="https://www.linkedin.com/in/paulbrighton/"
                >
                  LinkedIn
                </a>
              </li>
            </ul>
          </section>,
          <ContactForm />,
        ],
        "className": "o-contact__container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Contact",
          },
          "ref": null,
          "rendered": "Contact",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "c-header-divider",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Connect with me
              </h3>,
              <p>
                I am conveniently situated between Wakefield and Doncaster in Fitzwilliam, Wakefield and have access to the train station that connects to Leeds and Sheffield.
              </p>,
              <p>
                I will be actively looking for work as a web developer as soon as I finish my course at {Pro}Coders.
              </p>,
              <p>
                Feel free to contact me with any comments or inquires using the contact form or connect with me on social media.
              </p>,
              <ul>
                <li>
                  <a
                    href="https://twitter.com/paulbrighton_"
                  >
                    Twitter
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.facebook.com/paul.brighton.10"
                  >
                    Facebook
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.pinterest.co.uk/paulbrightonfda/"
                  >
                    Pinterest
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.linkedin.com/in/paulbrighton/"
                  >
                    LinkedIn
                  </a>
                </li>
              </ul>,
            ],
            "className": "c-contact-details",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Connect with me",
              },
              "ref": null,
              "rendered": "Connect with me",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "I am conveniently situated between Wakefield and Doncaster in Fitzwilliam, Wakefield and have access to the train station that connects to Leeds and Sheffield.",
              },
              "ref": null,
              "rendered": "I am conveniently situated between Wakefield and Doncaster in Fitzwilliam, Wakefield and have access to the train station that connects to Leeds and Sheffield.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "I will be actively looking for work as a web developer as soon as I finish my course at {Pro}Coders.",
              },
              "ref": null,
              "rendered": "I will be actively looking for work as a web developer as soon as I finish my course at {Pro}Coders.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Feel free to contact me with any comments or inquires using the contact form or connect with me on social media.",
              },
              "ref": null,
              "rendered": "Feel free to contact me with any comments or inquires using the contact form or connect with me on social media.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li>
                    <a
                      href="https://twitter.com/paulbrighton_"
                    >
                      Twitter
                    </a>
                  </li>,
                  <li>
                    <a
                      href="https://www.facebook.com/paul.brighton.10"
                    >
                      Facebook
                    </a>
                  </li>,
                  <li>
                    <a
                      href="https://www.pinterest.co.uk/paulbrightonfda/"
                    >
                      Pinterest
                    </a>
                  </li>,
                  <li>
                    <a
                      href="https://www.linkedin.com/in/paulbrighton/"
                    >
                      LinkedIn
                    </a>
                  </li>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <a
                      href="https://twitter.com/paulbrighton_"
                    >
                      Twitter
                    </a>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Twitter",
                      "href": "https://twitter.com/paulbrighton_",
                    },
                    "ref": null,
                    "rendered": "Twitter",
                    "type": "a",
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <a
                      href="https://www.facebook.com/paul.brighton.10"
                    >
                      Facebook
                    </a>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Facebook",
                      "href": "https://www.facebook.com/paul.brighton.10",
                    },
                    "ref": null,
                    "rendered": "Facebook",
                    "type": "a",
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <a
                      href="https://www.pinterest.co.uk/paulbrightonfda/"
                    >
                      Pinterest
                    </a>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Pinterest",
                      "href": "https://www.pinterest.co.uk/paulbrightonfda/",
                    },
                    "ref": null,
                    "rendered": "Pinterest",
                    "type": "a",
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <a
                      href="https://www.linkedin.com/in/paulbrighton/"
                    >
                      LinkedIn
                    </a>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "LinkedIn",
                      "href": "https://www.linkedin.com/in/paulbrighton/",
                    },
                    "ref": null,
                    "rendered": "LinkedIn",
                    "type": "a",
                  },
                  "type": "li",
                },
              ],
              "type": "ul",
            },
          ],
          "type": "section",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <div
          className="o-contact__container"
        >
          <h2>
            Contact
          </h2>
          <div
            className="c-header-divider"
          />
          <section
            className="c-contact-details"
          >
            <h3>
              Connect with me
            </h3>
            <p>
              I am conveniently situated between Wakefield and Doncaster in Fitzwilliam, Wakefield and have access to the train station that connects to Leeds and Sheffield.
            </p>
            <p>
              I will be actively looking for work as a web developer as soon as I finish my course at {Pro}Coders.
            </p>
            <p>
              Feel free to contact me with any comments or inquires using the contact form or connect with me on social media.
            </p>
            <ul>
              <li>
                <a
                  href="https://twitter.com/paulbrighton_"
                >
                  Twitter
                </a>
              </li>
              <li>
                <a
                  href="https://www.facebook.com/paul.brighton.10"
                >
                  Facebook
                </a>
              </li>
              <li>
                <a
                  href="https://www.pinterest.co.uk/paulbrightonfda/"
                >
                  Pinterest
                </a>
              </li>
              <li>
                <a
                  href="https://www.linkedin.com/in/paulbrighton/"
                >
                  LinkedIn
                </a>
              </li>
            </ul>
          </section>
          <ContactForm />
        </div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2>
              Contact
            </h2>,
            <div
              className="c-header-divider"
            />,
            <section
              className="c-contact-details"
            >
              <h3>
                Connect with me
              </h3>
              <p>
                I am conveniently situated between Wakefield and Doncaster in Fitzwilliam, Wakefield and have access to the train station that connects to Leeds and Sheffield.
              </p>
              <p>
                I will be actively looking for work as a web developer as soon as I finish my course at {Pro}Coders.
              </p>
              <p>
                Feel free to contact me with any comments or inquires using the contact form or connect with me on social media.
              </p>
              <ul>
                <li>
                  <a
                    href="https://twitter.com/paulbrighton_"
                  >
                    Twitter
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.facebook.com/paul.brighton.10"
                  >
                    Facebook
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.pinterest.co.uk/paulbrightonfda/"
                  >
                    Pinterest
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.linkedin.com/in/paulbrighton/"
                  >
                    LinkedIn
                  </a>
                </li>
              </ul>
            </section>,
            <ContactForm />,
          ],
          "className": "o-contact__container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Contact",
            },
            "ref": null,
            "rendered": "Contact",
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "c-header-divider",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3>
                  Connect with me
                </h3>,
                <p>
                  I am conveniently situated between Wakefield and Doncaster in Fitzwilliam, Wakefield and have access to the train station that connects to Leeds and Sheffield.
                </p>,
                <p>
                  I will be actively looking for work as a web developer as soon as I finish my course at {Pro}Coders.
                </p>,
                <p>
                  Feel free to contact me with any comments or inquires using the contact form or connect with me on social media.
                </p>,
                <ul>
                  <li>
                    <a
                      href="https://twitter.com/paulbrighton_"
                    >
                      Twitter
                    </a>
                  </li>
                  <li>
                    <a
                      href="https://www.facebook.com/paul.brighton.10"
                    >
                      Facebook
                    </a>
                  </li>
                  <li>
                    <a
                      href="https://www.pinterest.co.uk/paulbrightonfda/"
                    >
                      Pinterest
                    </a>
                  </li>
                  <li>
                    <a
                      href="https://www.linkedin.com/in/paulbrighton/"
                    >
                      LinkedIn
                    </a>
                  </li>
                </ul>,
              ],
              "className": "c-contact-details",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Connect with me",
                },
                "ref": null,
                "rendered": "Connect with me",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "I am conveniently situated between Wakefield and Doncaster in Fitzwilliam, Wakefield and have access to the train station that connects to Leeds and Sheffield.",
                },
                "ref": null,
                "rendered": "I am conveniently situated between Wakefield and Doncaster in Fitzwilliam, Wakefield and have access to the train station that connects to Leeds and Sheffield.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "I will be actively looking for work as a web developer as soon as I finish my course at {Pro}Coders.",
                },
                "ref": null,
                "rendered": "I will be actively looking for work as a web developer as soon as I finish my course at {Pro}Coders.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Feel free to contact me with any comments or inquires using the contact form or connect with me on social media.",
                },
                "ref": null,
                "rendered": "Feel free to contact me with any comments or inquires using the contact form or connect with me on social media.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <li>
                      <a
                        href="https://twitter.com/paulbrighton_"
                      >
                        Twitter
                      </a>
                    </li>,
                    <li>
                      <a
                        href="https://www.facebook.com/paul.brighton.10"
                      >
                        Facebook
                      </a>
                    </li>,
                    <li>
                      <a
                        href="https://www.pinterest.co.uk/paulbrightonfda/"
                      >
                        Pinterest
                      </a>
                    </li>,
                    <li>
                      <a
                        href="https://www.linkedin.com/in/paulbrighton/"
                      >
                        LinkedIn
                      </a>
                    </li>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        href="https://twitter.com/paulbrighton_"
                      >
                        Twitter
                      </a>,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Twitter",
                        "href": "https://twitter.com/paulbrighton_",
                      },
                      "ref": null,
                      "rendered": "Twitter",
                      "type": "a",
                    },
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        href="https://www.facebook.com/paul.brighton.10"
                      >
                        Facebook
                      </a>,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Facebook",
                        "href": "https://www.facebook.com/paul.brighton.10",
                      },
                      "ref": null,
                      "rendered": "Facebook",
                      "type": "a",
                    },
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        href="https://www.pinterest.co.uk/paulbrightonfda/"
                      >
                        Pinterest
                      </a>,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Pinterest",
                        "href": "https://www.pinterest.co.uk/paulbrightonfda/",
                      },
                      "ref": null,
                      "rendered": "Pinterest",
                      "type": "a",
                    },
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        href="https://www.linkedin.com/in/paulbrighton/"
                      >
                        LinkedIn
                      </a>,
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "LinkedIn",
                        "href": "https://www.linkedin.com/in/paulbrighton/",
                      },
                      "ref": null,
                      "rendered": "LinkedIn",
                      "type": "a",
                    },
                    "type": "li",
                  },
                ],
                "type": "ul",
              },
            ],
            "type": "section",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
